components/result-count-title/src/index.js

```diff
@@ -4,16 +4,19 @@ import styled from 'react-emotion';
 
 import ResultCount from '@govuk-frederic/result-count';
 
-const Wrapper = styled('div')({
-  display: 'inline-block',
-});
+const Wrapper = styled('div')(
+  {
+    display: 'flex',
+    flexWrap: 'nowrap',
+  },
+);
 
 const Title = styled('div')({
   display: 'inline-block',
   fontFamily: 'Roboto, Arial, sans-serif',
   fontSize: '15px',
   lineHeight: '1.25',
-  marginLeft: '10px',
+  padding: '2px',
```

Comment:
not sure why this has been added - will also affect how this component appears in other places where it's used

Reply:
Inside the Flexbox layout the title does seem to need a little bit of vertical padding, but perhaps `1px`, as the `span` beside it has. I've taken this off completely for now though - see what you think.